Add more information to clinical presentation subsection (#374)
greenelab/covid19-review: content/02.introduction.md

#### Clinical Presentation of COVID-19

> A great diversity of symptom profiles has been observed for COVID-19, although fever and cough are very common.  [@doi:10.1056/NEJMoa2002032]

Reviewer: ```suggestion
A great diversity of symptom profiles has been observed for COVID-19, although fever and cough are very common and were initially reported on admission in 43.8% and 63.7%, respectively.  [@doi:10.1056/NEJMoa2002032]  Recent data from the New York City region have showed variable rates of fever as a presenting symptom, ranging from 30.7% to 77.1% of patients [@doi:10.1001/jama.2020.6775][@doi:10.1056/NEJMc2010419].  Of note, 20.4% of patients presented with a low oxygen saturation (<90%) [@doi:10.1001/jama.2020.6775], while others have noted cough (79.4%), fever (77.1%), and dyspnea (56.5%)as the most common presenting symptoms [@doi:10.1056/NEJMc2010419].  The variability shown in the presenting symptoms of infection by SARS-CoV-2 makes it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate infection and should prompt isolation.
```

Author: Maybe if we are going to provide exact percentages, we might have to tell more about the samples from each study.  The percentages would be different in a sample in China, Seattle, or New York, even if the age range was similar.  Do you think so?  I was intentionally being vague about exact numbers in order not to get into that, but I think it's fine to go into more detail too.

Reviewer: That's fine with me.  I can remove the percentages so we don't get too specific.

Author: What do you think, @rando2 ?  Ordinarily I would think that more precise information was better.  But it is sometimes hard to decide.

Reviewer: My thought process was that the presenting symptoms should not be very dependent on location.  Ie. the percentage of patients presenting with fever shouldn't change if you're in China vs. Seattle vs. NYC (which we show is not correct and it does vary).  Outcomes, on the other hand, would be very dependent on location, available care, drug regimens, etc. and the specific percentages for those would need to be further explained (ironically, these are all somewhat close between various regions and are very poor overall).

Author: Come to think of it, I suppose that sometimes, the differences in populations might be part of an interesting story.  For instance, the kidney pathology paper (#325) mentioned that acute kidney damage seemed to be more of a problem in New York than in Seattle or China, perhaps due to more preexisting problems like diabetes.  Francesca Dominici at Harvard also has a tech report linking COVID-19 deaths to air pollution levels, and of course there is a lot of justified concern about racial disparities.  But I don't know where, if anywhere, those macro-scale issues would fit into this particular paper.

Reviewer: That gets very tricky and could really send us spiraling.  It's also incredibly hard to separate pre-existing conditions vs. racial disparities, as those are not mutually exclusive.  My institution's data (still under review, dragging on forever) suggests that it's more of the overall health picture of the patients rather than race that effect survival, but it's impossible to separate the two.  

I don't want this paper to get too dragged down with the clinical side (which is continually evolving) since it's more of an overarching review.

Author: That may be a good point -- maybe sometimes it's worse to say something simplistic than just to say nothing.  We won't be able to cite every existing COVID-19 paper anyway, since they are coming out faster than we could read them.

Reviewer: I almost want to keep the whole clinical section broad and somewhat simple.  

- This is how they present (fever, cough, etc)
- These are the complications (DVT, PE, stroke, end-organ failure)
- These are the outcomes (we can get specific in % mortality or keep it broad and say something along the lines of "significant mortality has been shown across populations and pandemic hot spots, including Wuhan, Seattle, and NYC"

Author: That seems like a good structure -- @rando2, does this seem reasonable?

Reviewer 2: This seems very reasonable to me! There is also always the option to add in more information as a new section down the road if it starts to seem like we need it. @scapone01 I edited your comment to move each sentence to a new line (which makes it a little easier to track edits), and I'll defer to @dziakj1 to commit it into the text!

Reviewer: @rando2 Thanks so much!  I'll keep that in mind with my next set of edits.  I'm going to sit down and try to do them all at once so it's more organized and only one suggested change vs. multiple smaller ones.

The symptoms of infection by SARS-CoV-2 can vary greatly, making it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate infection and should prompt isolation.
[@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment scores, as well as high levels of d-dimer. 
Mortality might be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing. 
[@doi:10/ggnxb3] reported that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
A later study reported that many hospitalized patients had radiographic findings such as ground-glass opacity and bilateral patchy shadowing, and most had lymphocytopenia. [@doi:10.1056/NEJMoa2002032]
COVID-19 can affect diverse body systems in addition to causing respiratory problems.  

> For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].

Reviewer: ```suggestion
For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].  COVID-19 has also been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 [@doi:10.1056/NEJMc2009787], and other thrombotic events including pulmonary embolism and deep vein thrombosis [@doi:10.1016/j.thromres.2020.04.013].  The mechanism behind these complications has been suggested to be related to coagulopathy with reports of antiphospholipid antidbodies present [@doi:10.1056/NEJMc2007575] and elevated levels of d-dimer and fibrinogen degradation products (FDP) elevated in deceased patients [@doi:10.1111/jth.14768]. 
```

Author: I am so glad that you are expanding this section!  I am not a medical expert and was just summarizing studies I had read.  I would defer to @rando2 but I think it would be great to be as detailed as possible.  
We also didn't say anything yet about pediatric symptoms and MIS/C or Kawasaki-like syndrome yet, as far as I know -- I feel like somebody should say something about that.  I could work on a rough outline but I am sure that @scapone01 would be much better qualified to discuss it intelligently.

Reviewer 2: @dziakj1 @scapone01 this seems like a good level of detail to include to me! I know both renal failure and antiphospholipid syndrome have at various point been factored into considerations for why HCQ might or might not be useful for COVID-19, so this is very interesting to see here as well. I would actually say, if you wouldn't mind explaining coagulopathy a little more and whether it is seen in other viral infections, that would also be interesting background!

Reviewer: @rando2 @dziakj1 Sounds good, I'll get to work on this.  I'll provide a little extra background, but don't want to send people down too much of a rabbit hole.  I'll also add in some more detail on both presenting symptoms and outcomes.  I'm going to do my best to not inundate you two with updates and do it in an efficient manner.  I apologize in advance, I'm a Github rookie.

Reviewer 2: @scapone01 that sounds great! Please let me know if you need any help, GitHub is a new platform for most people working on this! I tried to put together some info [here](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#reviewing-someones-pull-request) that might help

Reviewer: @rando2 Thank you, that page has been incredibly helpful.  I updated my comment here to add some additional information about viral infections and coagulopathy.  I can go into more detail about the specifics of the coagulation cascade if you think that's relevant.

Reviewer 2: @scapone01 this is super interesting!!! I made some superficial edits (just split the lines apart). @dziakj1 does this look good to you?

Author: @scapone01  There were a few places where I wasn't sure what you meant.  Could you be clearer about "were initially reported to be the two most common symptoms on admission"?  I don't know what "initially" means here, because I'm not confident about whether the article we are citing is the very first article ever on COVID-19.  Also, " on room air" might not be obvious to all readers;  could it be "when not on artificial oxygen supplementation or ventilation"?  I also don't know what you mean by "variable rates of fever as a presenting symptom."  Do you mean that there are different rates at different times, or different places, or for different people?  The context of the sentence suggests that only New York is being discussed in that place, so I don't know what is supposed to vary.  Thanks!

Reviewer: Yea this one needs some work.  All very good points.  I'll edit tonight and try to clarify each of those points and make it more concise and cleaned up.

Reviewer: @dziakj1 How do you feel about the other additions regarding complications?

#### Clinical Presentation of COVID-19

> A great diversity of symptom profiles has been observed for COVID-19, although fever and cough are very common and were initially reported to be the two most common symptoms on admission [@doi:10.1056/NEJMoa2002032].

Reviewer: ```suggestion
A great diversity of symptom profiles has been observed for COVID-19, although a large study from Wuhan, China suggests fever and cough as the two most common symptoms on admission [@doi:10.1056/NEJMoa2002032].
```

> Recent data from the New York City region have showed variable rates of fever as a presenting symptom [@doi:10.1001/jama.2020.6775][@doi:10.1056/NEJMc2010419].

Reviewer: ```suggestion
But recent data from the New York City region have shown variable rates of fever as a presenting symptom, leading to a less defined incidence across populations [@doi:10.1001/jama.2020.6775] [@doi:10.1056/NEJMc2010419].  
These differences are present when comparing between both institutions in similar locations and between different regions experiencing COVID-19 outbreaks, leading to conflicting reports of the frequency of fever as a presenting symptom for patients upon hospital admission.
```

> Of note, low oxygen saturation (<90% on room air) was noted in a significant percentage of patients on presentation [@doi:10.1001/jama.2020.6775], while others have noted cough, fever, and dyspnea as the most common presenting symptoms [@doi:10.1056/NEJMc2010419].

Reviewer: ```suggestion
Of note, low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) was noted in a significant percentage of patients on presentation [@doi:10.1001/jama.2020.6775], while others have noted cough, fever, and dyspnea as the most common presenting symptoms [@doi:10.1056/NEJMc2010419].
```

Mortality might be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing. 
[@doi:10/ggnxb3] reported that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
A later study reported that many hospitalized patients had radiographic findings such as ground-glass opacity and bilateral patchy shadowing, and most had lymphocytopenia. [@doi:10.1056/NEJMoa2002032]
COVID-19 can affect diverse body systems in addition to causing respiratory problems.  
For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].
COVID-19 has also been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 [@doi:10.1056/NEJMc2009787], and other thrombotic events including pulmonary embolism and deep vein thrombosis [@doi:10.1016/j.thromres.2020.04.013].

> The mechanism behind these complications has been suggested to be related to coagulopathy with reports of antiphospholipid antidbodies present [@doi:10.1056/NEJMc2007575] and elevated levels of d-dimer and fibrinogen degradation products (FDP) elevated in deceased patients [@doi:10.1111/jth.14768].

Reviewer: ```suggestion
The mechanism behind these complications has been suggested to be related to coagulopathy with reports of antiphospholipid antibodies present [@doi:10.1056/NEJMc2007575] and elevated levels of d-dimer and fibrinogen degradation products (FDP) elevated in deceased patients [@doi:10.1111/jth.14768].
```

Reviewer: Sorry, typo.

Other viral infections have been associated with coagulation defects, notably the coronavirus Severe Acute Respiratory Syndrome (SARS) which was also noted to lead to disseminated intravascular coagulation (DIC) and was associated with both pulmonary embolism and deep vein thrombosis [@doi:10.1002/jmv.23354].
A wide range of viral infections can effect the coagulation cascade.
The mechanism behind these insults has been suggested to be related to inflammation-induced increases in von Willebrand factor, leading to a pro-coagulative state [@doi:10.1002/jmv.23354].

> 

Reviewer: ```suggestion
Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].  
Once hospitalized, outcomes have generally been poor with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032] [@doi:10.1001/jama.2020.2648] [@doi:10.1001/jama.2020.4031].

```

Reviewer: Should I go ahead and talk about mortality here or is that too much?   We could end up pulling in lots of different mortality statistics based on differing populations (ventilated patients vs. all hospitalized vs. "critically ill") so I want to make sure we're as clear as possible.

The mechanism behind these complications has been suggested to be related to coagulopathy with reports of antiphospholipid antibodies present [@doi:10.1056/NEJMc2007575] and elevated levels of d-dimer and fibrinogen degradation products (FDP) elevated in deceased patients [@doi:10.1111/jth.14768].

Other viral infections have been associated with coagulation defects, notably the coronavirus Severe Acute Respiratory Syndrome (SARS) which was also noted to lead to disseminated intravascular coagulation (DIC) and was associated with both pulmonary embolism and deep vein thrombosis [@doi:10.1002/jmv.23354].
A wide range of viral infections can effect the coagulation cascade.
The mechanism behind these insults has been suggested to be related to inflammation-induced increases in von Willebrand factor, leading to a pro-coagulative state [@doi:10.1002/jmv.23354].
Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].  

> Once hospitalized, outcomes have generally been poor with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032] [@doi:10.1001/jama.2020.2648] [@doi:10.1001/jama.2020.4031].

Reviewer 2: ```suggestion
Outcomes in patients that are hospitalized have generally been poor with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
```

> Pediatric infections are also a concern.

Reviewer 2: ```suggestion
Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
```

@doi:10.1001/jamapediatrs.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.

> Most of the children reviewed had mild (fever, dry cough, fatigue, nasal congestion and/or runny nose) or no symptoms. 

Reviewer 2: ```suggestion
Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic. 
```

Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
Radiologic findings were not always mentioned, but when mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions.
The results of COVID-19 infection can vary greatly among pediatric patients as well as adults. 

> In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069][@doi:10.1001/jama.2020.10369] [10.1016/j.ajem.2020.05.117]. 

Reviewer 2: ```suggestion
In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117]. 
```

> Heart failure can occur [@doi:10.1161/CIRCULATIONAHA.120.048360].

Reviewer 2: ```suggestion
Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].
```
